app/routes/overview/components/NextEvent.css

```diff
@@ -12,6 +12,13 @@
   margin: 0;
 }
 
+.info {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
```

Comment:
Preferable use the `<Flex />` component for this, especially if you end up getting rid of the margin.

Reply:
Fixed